lib/rules/cli/4.0.yaml

```diff
@@ -1012,6 +1012,7 @@
     :reference_policy: --reference-policy=<value>
     :scheduled: --scheduled=<value>
     :source_type: --source=<value>
+    :n: --namespace=<value>
```

Comment:
@xiuwang, I don't think you need this because `n` and `namespace` are both under the `global:` section.  https://github.com/openshift/verification-tests/blob/master/lib/rules/cli/4.0.yaml#L14-L15

Reply:
Peter, thanks, I have removed this option.